velox/common/file/File.h

```diff
@@ -303,6 +312,7 @@ class LocalReadFile final : public ReadFile {
  private:
   void preadInternal(uint64_t offset, uint64_t length, char* pos) const;
 
+  folly::Executor* const executor_;
   std::string path_;
```

Comment:
```
const std::string path_;
```

Reply:
It is not const, this is two constructors, one input argument is path, one argument of another constructor is fd